dali/pipeline/executor/executor.cc

```diff
@@ -348,30 +348,35 @@ void Executor<WorkspacePolicy, QueuePolicy>::RunHelper(OpNode &op_node, Workspac
     if (had_empty_layout) empty_layout_in_idxs.push_back(i);
   }
 
-  if (op.Setup(output_desc, ws)) {
-    DALI_ENFORCE(
-        static_cast<size_t>(ws.NumOutput()) == output_desc.size(),
-        "Operator::Setup returned shape and type information for mismatched number of outputs");
-    DALI_ENFORCE(op.CanInferOutputs(),
-                 "Operator::Setup returned true indicating that it successfully calculated shape "
-                 "and type information for Operator outputs. In that case CanInferOutputs should "
-                 "always return true.");
-    for (int i = 0; i < ws.NumOutput(); i++) {
-      auto &desc = output_desc[i];
-      if (ws.template OutputIsType<CPUBackend>(i)) {
-        ws.template Output<CPUBackend>(i).Resize(desc.shape, desc.type);
-      } else {
-        ws.template Output<GPUBackend>(i).Resize(desc.shape, desc.type);
+  {
+    DomainTimeRange tr("Setup");
```

Comment:
This will also measure the allocation of the outputs - we could argue it's a part of setup, but is it really? Just asking to clarify.
Also - will this name be somehow connected to the specific operator which is being set up?

Reply:
> This will also measure the allocation of the outputs - we could argue it's a part of setup, but is it really? Just asking to clarify.

No preferences. I can extract and measure only setup itself. The gap in the profile between setup and run would be the allocation.

> Also - will this name be somehow connected to the specific operator which is being set up?

This is how it looks like:
![image](https://user-images.githubusercontent.com/39967756/169974770-c6a984ae-d40d-4c56-a852-84719f05eb28.png)
It is inside the Operator range. But I can add `[DALI]` to the name as we do in other ranges.

Reply:
After changes 
![image](https://user-images.githubusercontent.com/39967756/169986755-bb60af72-3f39-48b9-8249-9c15e1ec7408.png)